### PR TITLE
Created a data generator script

### PR DESCRIPTION
**Pull request by JonBaltz**

### CODE REVIEW

**Comment by JonBaltz** [comment]
@cranjis-mcbasketball @anthony-ha @jacobwilson41

File: server/seeds/mysqlSeed.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  database: 'media'
+});
+
+connection.connect(err => {
+  if (err) {
+    console.log(err)
+  }
+});
+
+for (let i = 0; i < 100; i++) {
```
Comment (jacobwilson41) [inline server/seeds/mysqlSeed.js:15]: why only 100?  Where does the 10,000,000 come into play?

File: server/seeds/generator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const songURL = ['https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-allthat.mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-badass+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-funnysong+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-hipjazz+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-jazzcomedy+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-moose+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-perception+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-retrosoul+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-sexy+(1).mp3', 'https://sdc-media-player.s3.us-east-2.amazonaws.com/songs2/bensound-theelevatorbossanova+(1).mp3'];
```
Comment (jacobwilson41) [inline server/seeds/generator.js:9]: Are these urls to assets you personally added to S3?

Comment (JonBaltz) [reply server/seeds/generator.js:9]: Yes they are

**Comment by JonBaltz** [comment]
This wasn't the end, I dont know why i created a pull request for this.

**Pull request closed without merging**